peer/brontide_test.go

```diff
@@ -971,14 +976,15 @@ func TestStaticRemoteDowngrade(t *testing.T) {
 				},
 			}
 
-			var b bytes.Buffer
-			_, err := lnwire.WriteMessage(&b, test.expectedInit, 0)
-			require.NoError(t, err)
-
 			// Send our init message, assert that we write our expected message
 			// and shutdown our write pool.
 			require.NoError(t, p.sendInitMsg(test.legacy))
-			mockConn.assertWrite(b.Bytes())
+
+			// Check that mockConn sent out the expected message.
+			msg, err := getMessage(mockConn)
+			require.NoError(t, err)
+			require.Equal(t, msg, test.expectedInit)
```

Comment:
nit: the recommended order is `reqire.Equal(t, expected, got)`

Reply:
changed in the commit it's introduced, a later commit removes it